lib/expressions.js

```diff
@@ -2,6 +2,24 @@
 
 const OPERATORS = require('./operators.js');
 
+const BOOL = {
+  t: true,
+  nil: false,
+};
+
+class BooleanExpression {
+  constructor(value) {
+    if (!(value in BOOL)) {
+      throw new Error(`Unknown boolean value: ${value}`);
+    }
+    this.value = value;
```

Comment:
We can store `value` as javascript boolean type, receiving either js boolean or t/nil